features/step_definitions/ssp_steps.rb

```diff
@@ -99,8 +120,8 @@
 
 Then /^I am taken to the '(.*)' page$/ do |page_name|
   find('h1').should have_content(/#{page_name}/i)
-  store.framework_name = URI.parse(current_url).path.split('frameworks/').last.split('/').first
-  store.service_type = URI.parse(current_url).path.split('submissions/').last.split('/').first
+  #store.framework_name = URI.parse(current_url).path.split('frameworks/').last.split('/').first
```

Comment:
It would be good to delete these commented out lines unless we think we might need them again soon.